ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a summary of the issue in the Title above -->
```

Comment:
I think because he did a rebase and modified a commit that is already on `dev`. So the PR needs to remove that edit, (maybe adding the change as a separate commit) and rebase on the dev.

Reply:
I'll take a look again after the rebase, if someone @'s me

Reply:
You're right, I started working off my old branch directly.
I've since rebased from `latest dev`.